### DISCUSSION TITLE
global settings in Modrinth app

### DISCUSSION THREAD

**Discussion by spydragon** [modrinth/.github#202] [Feature requests]
I have used a couple of different third-party launchers and I have seen one issue I would love to have fixed in all of them.

If you have a setup that differs from the default, for example, max brightness, shift as sprint (for some reason), no auto jump, sensitivity hyperspeed, FOV of 100, and so on, this will not stay the same if you switch into a different instance. and if you create a new instance, you have to spend a bit of time trying to redo everything.

this is not a big issue, but it is one of those QOL things I would find greatly convenient.

To fix this issue I suggest a new tab in the settings of Modrinth. this tab would list the most common settings in Minecraft, like movement controls, brightness, clouds, sounds, quality, and so on. when these settings are changed, the settings will then be transferred to every Minecraft instance.

Do I think this will be easy? no. I think this might be a bit difficult. it all depends on how much 1). Mojang hasn't changed how settings are stored in different versions of Minecraft. and 2). mods change how settings are stored.

But I also know nothing about how to code something like this, so for all I know it's easy, or even more difficult. either way, if the Modrinth app was made to do this (even if it were difficult to do), I think it would grow even more popular quite quickly.

I would like to hear everyone's thoughts, is this a good idea? if so, please upvote. if not, why? any suggestions to make it better?

PS: I didn't see a feature request page for the Modrinth app, but I did see it for 'Modrinth' which included a link to the app. If this is in the wrong place I am sorry and would like to know where I should put this request.

**Comment by Arthurmeade12** [comment]
https://modrinth.com/mod/shared-resources sounds like what you want @spydragon

**Reply by spydragon** [reply]
That would work if I was using the official Minecraft launcher, but it wouldn’t work for Modrinth, multiMC or curse forge. In the case of modrinth and multiMC it’s because the launcher is third party, in the case of curse forge, it’s because each modpack is a separate launcher (which I think is dumb but that’s not the point here)

I could be wrong, and this mod might do what I want, but then I would need that mod in every instance I play and that’s still inconvenient.

**Reply by Arthurmeade12** [reply]
The mod explicitly recommends itself for multimc. Shared Resources let's you set an arbitrary global directory for resource packs, options.txt, etc. So it would in fact be compatible with every launcher. You would however need to put it into every instance you play, which is a nuisance.

**Reply by spydragon** [reply]
True, I read the description of the mod and I see that it is what I want. But I am glad that we agree that having to add it to every instance is annoying.

but the mod is also only for fabric. While fabric is (in my opinion) the best option for mods, there are some mod packs that I play that are forge based. And I (or the average individual) don’t know how to combine both systems together.

it would be so nice if it were integrated into the modrinth app.

**Comment by Gabriel-Science** [comment]
This is by far the most annoying thing with instances.
